Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -36,7 +37,7 @@ const cardContainer = (show: boolean, ref?: HTMLDivElement | null) => () => {
     border-left: 4px solid #0078d4;
     background: white;
     box-shadow: 0 6.4px 14.4px 0 rgba(0, 0, 0, 0.132), 0 1.2px 3.6px 0 rgba(0, 0, 0, 0.108);
-    min-width: 340px;
+    width: 340px;
```

Comment:
Why did you change this from `min-width` to `width`?

Reply:
This is in order to keep the width of a card consistent when the text inside needs to wrap to multiple lines. They fit into the sidebar that way.
![image](https://user-images.githubusercontent.com/61990921/123163678-d8700d80-d426-11eb-8256-ce885f806f34.png)